config/sources/s500.conf

```diff
@@ -13,18 +13,19 @@ GOVERNOR=interactive
 
 case $BOARD in
 	guitar)
-	UBOOT_FILES="$SRC/lib/bin/bootloader-guitar.bin u-boot-dtb.img"
+	BOOTLOADER_BIN="s500-bootloader-guitar.bin"
+	UBOOT_FILES="$SRC/lib/bin/$BOOTLOADER_BIN u-boot-dtb.img"
 	;;
 
 	roseapple)
-	UBOOT_FILES="$SRC/lib/bin/bootloader-roseapple.bin u-boot-dtb.img"
+	BOOTLOADER_BIN="s500-bootloader-roseapple.bin"
+	UBOOT_FILES="$SRC/lib/bin/$BOOTLOADER_BIN u-boot-dtb.img"
 	;;
 esac
 
 write_uboot_platform()
 {
-	[ -f "$1/s500-bootloader-guitar.bin" ] && dd if=$1/bootloader-guitar.bin of=$2 bs=512 seek=4097 conv=fsync > /dev/null 2>&1
-	[ -f "$1/s500-bootloader-roseapple.bin" ] && dd if=$1/bootloader-roseapple.bin of=$2 bs=512 seek=4097 conv=fsync > /dev/null 2>&1
+	dd if=$1/$BOOTLOADER_BIN of=$2 bs=512 seek=4097 conv=fsync > /dev/null 2>&1
```

Comment:
I'm sure that in current state updading u-boot package won't be possible.

> Compiled 2 test images for rose and guitar and both have no bootloader!

Did you fix file names in `dd` coomand (by adding `s500-` prefix)? I'm sorry that I didn't notice that Igor [didn't change](https://github.com/igorpecovnik/lib/commit/6664055a682a8075c3d1cd27d6b001e35a0eaf0a) file name in installation function.